listenbrainz/spark/handlers.py

```diff
@@ -0,0 +1,15 @@
+import listenbrainz.db.user as db_user
```

Comment:
This file feels rather bare without a single comment in it -- maybe add a comment near the top describing the point of these handlers?

Reply:
Added a docstring for the file and for the handler function